operator/Makefile

```diff
@@ -72,6 +72,15 @@ ifneq (Version: v$(CONTROLLER_GEN_VERSION), $(shell $(CONTROLLER_GEN) --version)
 	GOFLAGS='' go install sigs.k8s.io/controller-tools/cmd/controller-gen@v$(CONTROLLER_GEN_VERSION)
 endif
 
+# Install CRDs into a cluster
+.PHONY: install
+install: manifests
```

Comment:
I think these are existing from our internal repo (?), so we probably want to add them as-is and then make them better.

Some ideas for how to make them better:

* Use server-side apply
* Maybe rename install to install-crds? (and same for uninstall)
* If deploy relies on docker-build, maybe we should make it a makefile dependency :-)

But these are nits and I vote to merge and iterate!

Reply:
Yeah these rules are from internal Makefile which I used often when working on the operator code.